images/daemon/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM op5com/merlin-base:latest
+
+ENV NAEMON_CFG=/etc/naemon/ MERLIN_CFG=/etc/merlin/ \
+    MERLIN_SOCK=/var/lib/merlin/ipc.sock \
+    NAEMON_UID=1001 NAEMON_GID=0
+
+COPY shared_files/install.sh /usr/local/bin/install.sh
+COPY daemon/default_config /usr/local/etc/merlin/default_config
+RUN bash /usr/local/bin/install.sh
+# Get rid of default merlin.conf, we'll replace by the default_config
+# during the entry script
+#RUN rm $MERLIN_CFG/merlin.conf
```

Comment:
Maybe remove if it really is a comment?

Reply:
Thanks, done. It was moved to install.sh.